documentation/Pipfile

```diff
@@ -4,7 +4,6 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-sphinxcontrib-mermaid = "*"
 
 [dev-packages]
 sphinx = "*"
```

Comment:
Curious that all the documentation packages are in the dev section.

Reply:
The build output are static pages so everything is a dev dependency.